### ISSUE TITLE
Faceted browser doesn't display some datatypes

### ISSUE THREAD

**Issue by IS4Code** [openlink/virtuoso-opensource#937]
The describe service seems to turn most datatypes like `xsd:unsignedByte` or even `xsd:duration` into just `xsd:integer`, and the rest, like `rdf:HTML` doesn't even display, when browsing the properties of resources. I can however confirm with SPARQL that the original datatypes are there.

Version 07.20.3230

**Comment by TallTed** [comment]
Please ensure that all your VAD packages are up to date.  Prebuilt binaries from Dec 2018 are [here](http://download3.openlinksw.com/index.html?prefix=uda/vad-vos-packages/7.2/).

At the moment, you'd be better off building your own VADs (along with the latest Virtuoso binary!) from the latest code in [the `develop/7` branch](https://github.com/openlink/virtuoso-opensource/commits/develop/7).

If you find the issue persists with builds from the latest codebase, it would be helpful if you could provide a link to a running instance where the issue is visible, along with step-by-step recreation instructions.

**Comment by IS4Code** [comment]
I was using a Docker package, but I will try to compile it. In the meantime, I can provide an example however: [incorrect](https://albion.data.is4.site/describe/?url=http%3A%2F%2Falbion.data.is4.site%2Flang%2FTerran), [correct](https://albion.data.is4.site/sparql?query=define%20sql%3Adescribe-mode%20%22CBD%22%20%20DESCRIBE%20%3Chttp%3A%2F%2Falbion.data.is4.site%2Flang%2FTerran%3E&output=text%2Fplain).

**Comment by TallTed** [comment]
I'm sorry; there's a lot of data on both of your example pages, and I can't quickly see the issue you're describing. Can you point me more explicitly to where the issue is on the "incorrect" page? I trust this will also point me more explicitly to where the issue is not, on the "correct" page.

**Comment by IS4Code** [comment]
Sorry, here it is:
![image](https://user-images.githubusercontent.com/1337208/105749985-74631f00-5f44-11eb-8b7f-b0e845aa612f.png)
```html
<span property="rpg:enumValue" xmlns:rpg="http://rpg.data.is4.site/">0</span> <small>(<a href="http://www.w3.org/2001/XMLSchema#integer">xsd:integer</a>)</small></span>
```
While in the SPARQL result the value is `"0"^^<http://www.w3.org/2001/XMLSchema#unsignedByte>`.

**Comment by TallTed** [comment]
@pkleef @timhaynesopenlink @HughWilliams 

Any thoughts?

**Comment by openlink** [comment]
I pushed a fix to the develop/7 branch

**Comment by TallTed** [comment]
@openlink - When will that fix be available in a pre-built VOS 7.2 VAD and/or binary?
